app/javascript/css/components/track.css

```diff
@@ -1,17 +1,36 @@
 @import "../styles";
 
 .c-track {
-    @apply border-black p-24 bg-white shadow-1 rounded-8 flex;
+    @apply flex items-start;
+    @apply border-black p-24 bg-white shadow-1 rounded-8;
 
     & .c-track-icon {
         width: 57px;
         @apply mr-16;
     }
-    & .title {
-        @apply text-h3-mono mb-8;
-    }
+    & .info {
+        @apply flex-grow;
+
+        & .title {
+            @apply mb-8;
+            @apply text-h3-mono mb-8 text-black;
+        }
 
-    & .counts {
-        @apply flex font-medium text-16 gap-12 text-textLight;
+        & .counts {
+            @apply flex gap-12 mb-12;
+            @apply font-medium text-16;
```

Comment:
I'm following the principle of having one line for layout and a separate line for styling.